jit/v2/JIT_v2.0.0.yaml

```diff
@@ -305,6 +624,102 @@ paths:
           headers:
             API-Version:
               $ref: '#/components/headers/API-Version'
+        '400':
+          description: |
+            In case omitting a `Port Call` does not schema validate a `400` (Bad Request) is returned
```

Comment:
In case omitting a 'Port Call' fails schema validation a 400 (Bad Request) is returned